src/platforms/dialogflow/DialogflowEvent.ts

```diff
@@ -58,30 +63,22 @@ export class DialogflowEvent extends VoxaEvent {
       type: "IntentRequest",
     };
 
-    this.intent = new DialogflowIntent(this.google.conv);
+    this.intent = new DialogflowIntent(this.dialogflow.conv);
   }
 
-  public async verifyProfile(): Promise<TokenPayload | undefined> {
-    const client = new OAuth2Client(this.platform.config.clientId);
-    const payload:
-      | TokenPayload
-      | undefined = await this.google.conv.user._verifyProfile(
-      client,
-      this.platform.config.clientId,
-    );
-
-    return payload;
+  public async getUserInformation(): Promise<any> {
+    return undefined;
   }
 
   protected initSession(): void {
-    this.google = {
+    this.dialogflow = {
```

Comment:
didn't we decide `this.google` was gonna be a deprecated property that  proxied to `this.dialogflow` ? I don't see that

Reply:
@armonge at the top of the class you'll find this
```
// DEPRECATED: INSTEAD USE dialogflow
  get google() {
    return this.dialogflow;
  }
```